PAWS/Reminders/Scheduler+Reminders.swift

```diff
@@ -0,0 +1,30 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SpeziScheduler
+
+
+extension Scheduler {
+    func scheduleReminders(time: DateComponents) async throws {
+        // We discard all other date components:
+        let hours = (time.hour.map { [$0] }) ?? []
+        let minutes = (time.minute.map { [$0] }) ?? []
```

Comment:
do we really need the parentheses here?

Reply:
Unfortunately yes, it doesn't compile on Swift 6.0, only Swift 6.1: https://github.com/StanfordBDHG/PediatricAppleWatchStudy/pull/85/commits/b545329a55da4aecac566382ee0c2e98c741c336